### PR TITLE
Switch to kaniko oci builder

### PR DESCRIPTION
**Pull request by ialidzhikov**
/kind enhancement

See https://github.com/gardener/gardener-extension-os-coreos/pull/19 and https://github.com/gardener/gardener-extension-os-coreos/pull/20 for more details.

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-FROM eu.gcr.io/gardener-project/3rd/golang:1.15.5 AS builder
+FROM golang:1.15.8 AS builder
 
 WORKDIR /go/src/github.com/gardener/gardener-extension-provider-aws
 COPY . .
 RUN make install
 
 ############# base
-FROM eu.gcr.io/gardener-project/3rd/alpine:3.12.3 AS base
+FROM alpine:3.12.3 AS base
```
Comment (vpnachev) [inline Dockerfile:9]:
```suggestion
FROM alpine:3.13.2 AS base
```

**Review by vpnachev — APPROVED** [review]
/lgtm

While you are touching the alpine image, could you also update it to the latest version :wink:

**Review by vpnachev — APPROVED** [review]
/lgtm

**Comment by ialidzhikov** [comment]
@rfranzke , can you help again with making the `concourse-ci/build_oci_image_gardener-extension-provider-aws` check required? Thanks in advance!

**Comment by rfranzke** [comment]
@ialidzhikov done :)

**Pull request merged**